dashboard/app/assets/stylesheets/application.scss

```diff
@@ -2730,3 +2730,27 @@ code {
     }
   }
 }
+
+.info-box-teal {
+  border: 1px solid $teal;
+  height: 46px;
+  margin-bottom: 10px;
+  .left {
```

Comment:
The scoping of this CSS to `.info-box-teal` is good.  The one thing to check is that we don't have any other more-globally declared `.left` that affects this component, though if it's looking okay then it's probably fine.